packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -562,8 +562,12 @@ export function getLastFocusableElement(element) {
     return null;
   }
 
+  if (element.tagName === "SLOT" && /** @type {HTMLSlotElement} */(element).assignedElements().length) {
+    return getLastFocusableElementFromSlot(/** @type {HTMLSlotElement} */(element))
+  }
```

Comment:
Not sure I like the inline typing comment :D Is there a way to not need that?

Reply:
inline typing removed 👌